_includes/whois-search.html

```diff
@@ -0,0 +1,212 @@
+                `).join("")}
+            </div>
+        `
+        return createCard(cardBody, "Nameservers");
+    }
+
+    function renderRdapNotices(notices=[], searchResultsId="rdap-search-results") {
+
+        const cardBody = `
+            <div class="usa-card__body">
+                ${notices.map(notice => `
+                    <h3>${notice.title ? notice.title : "Notice"}</h3>
+                    <p class="usa-card--details-body">${notice.description.join(" ")}</p>
+                `).join("")}
+            </div>
+        `
+        return createCard(cardBody, "Notices");
+    }
+
+    function searchRdapData(searchId="rdap-search", searchResultsId="rdap-search-results") {
+        event.preventDefault();
+
+        const rdapBaseUrl = 'https://rdap.cloudflareregistry.com/rdap/domain/';
+
+        let domain = document.getElementById(searchId).value
+
+        if (domain === null || domain.length === 0) {
+            render_result(searchResultsId, ``);
+            return;
+        }
+
+        if (!domain.endsWith('.gov')) {
+            let errorMessage = `
+                <div class="usa-alert usa-alert--error usa-alert--slim" role="alert">
+                    <div class="usa-alert__body  display-flex flex-column flex-align-center">
+                        <p class="usa-alert__text">
+                            You can only search domain names that end in .gov
+                        </p>
+                    </div>
+                </div>
+            `
+            render_result(searchResultsId, errorMessage);
+            return;
+        }
+
+        let rdapSearchEndpoint = new URL("", rdapBaseUrl + domain);
+        fetch(rdapSearchEndpoint).then(function(reply) {
+            return reply.json()
+        }).then(function(response) {
+            if (response.errorCode) {
+                let errorMessage = ``
+                if (response.errorCode == 404) {
+                    errorMessage = `
+                        <div class="usa-alert usa-alert--info usa-alert--slim">
+                            <div class="usa-alert__body">
+                                <p class="usa-alert__text">
+                                    <b>${domain}</b> isn't a registered domain.
+                                </p>
+                            </div>
+                        </div>
+                    `
+                } else {
+                    errorMessage = `
+                        <div class="usa-alert usa-alert--info usa-alert--slim">
+                            <div class="usa-alert__body">
+                                <p class="usa-alert__text">
+                                    Error looking up RDAP: ${response.description.join(" ")}
+                                </p>
+                            </div>
+                        </div>
+                    `
+                }
+                
+                render_result(searchResultsId, errorMessage, append=false);
```

Comment:
Append is false by default isn't it?

Reply:
It's true by default but I also feel like from the way we may use it in the future, it may make more sense to make it false by default. @zandercymatics I believe you may have made this method (correct me if I'm wrong) so was curious if there's any reason we'd want to default append to one or the other?